styles/main.scss

```diff
@@ -62,13 +63,28 @@ header nav {
 
 .main-container {
   margin: 0 auto;
-}
+  max-width: 90%;
 
-header#main-header {
-  padding-top: 40px;
+  h1 {
+    margin-top: 100pxf;
```

Comment:
`pxf` seems like a typo and does not do anything for me in firefox (I also can't find docs)